Scripts/parameters/assignment.py

```diff
@@ -183,6 +183,8 @@
 }
 # Default distance unit cost [eur/km]
 dist_unit_cost = 0.12
+# Default speed for trucks and trailer trucks [min/km]
```

Comment:
Maybe a bit confusing to talk about "speed", when this describes the trade-off between time and distance.